### ISSUE TITLE
[🐛] Get all challenges done on the screen  displayed- [Forged Feedback - Broken Access Control (Learning section) ]

### ISSUE THREAD

**Issue by AJ1062910** [juice-shop/juice-shop#1716]
# :bug: Bug report

## Description
When go into the Broken Access Control (Learning section) , and do the challenge it still an issue : 

![hackerman](https://user-images.githubusercontent.com/81747871/139051835-b35b8085-9a3d-445c-a799-bfba4f4e4b3c.gif)

As we can see I get all the message of successfully done all challenges 

### Additional Information
I let you guess how to reproduce this bug ;)

**Comment by bkimminich** [comment]
That's not a bug, that's a feature. This is what probably happened:

1. The https://demo.owasp-juice.shop Heroku instance (which we link to from https://owasp.org/www-project-juice-shop/#div-learning) went down at some point
2. It went back up after a short time as that is what Heroku luckily does automatically
3. You came along via the https://demo.owasp-juice.shop/#/hacking-instructor?challenge=Forged%20Feedback link as the first visitor after the restart (or: you were already on that page when the restart happened)
4. It restored the hacking progress from your browser cookie `continueCode` (or: from someone elses who happened to be on the page during restart)
5. This triggers the notifications of previously solved challenges

TL;DR: This happens when Juice Shop instances are used by multiple people. It's a design decision not to handle challenge tracking per user session but per instance. The demo instance is of course shared by many users, which is why things like this occur sometimes.
